conditionalsassignment3.js

```diff
@@ -0,0 +1,17 @@
+let machineActive = true;
+let pwd = 'cheese';
+
+let machineResult;
+let pwdResult;
+
+if (machineActive === true) {
+    machineResult = 'Machine switched ON';
+    console.log(machineResult);
+pwd === 'cheese' ? pwdResult = 'Password successful' : pwdResult ='Password Unsuccessful';
```

Comment:
The better way to write this would be

pwdResult = pwd === "cheese" ? "Password successful" : "Password Unsuccessful";